.github/workflows/ai-codereview.yml

```diff
@@ -12,32 +17,48 @@ jobs:
         with:
           repository: Mirantis/ai-ci-scripts
           path: ai-ci-scripts
+      - name: Get all changed golang files
+        id: changed-golang-files
+        uses: tj-actions/changed-files@v45
```

Comment:
As discussed previously and because of the [security breach](https://www.stepsecurity.io/blog/harden-runner-detection-tj-actions-changed-files-action-is-compromised), could we depend on a revision rather than the version? Something similar to the `tj-actions/changed-files@<revision> # v45`

Reply:
Why not just write inline bash script for that? Why depending on some 3rd party actions that already have bad security history?

Reply:
That's also okay. The previous version didn't come with a proper collection of changed files through multiple commits. It does not matter.